scripts/python-publish.sh

```diff
@@ -1,9 +1,6 @@
 #!/bin/bash
 set -e 
 
-# Build python source dist
-python setup.py sdist 
-
```

Comment:
This script was meant to publish, not to build. Separating that logic to avoid any confusion.